### PR TITLE
Fixed empty line number issue

### PR DESCRIPTION
**Pull request by sunilhari**
#11 Empty console.log() prints line number.

Omitted the lines from `result` array which only have line numbers.

### CODE REVIEW

**Comment by sallar** [comment]
@sunilhari your editor has re-formatted this file and changed the indention to 4. I think you need to revert the file and do your change again.

File: src/node_modules/editor/index.js [hunk 1]
```diff
@@ -85,121 +84,122 @@ async function render (s) {
...
-  })
+        return output.setValue([message, stack].join('\n'))
+    }
 
-  if (window.localStorage.matchingLines) {
-    const last = result[result.length - 1].line
-    output.setValue(Array(last || 1).fill('\n').join(''))
+    if (result.length === 0) {
+        return output.setValue('')
+    }
```
Comment (heapwolf) [inline src/node_modules/editor/index.js]: LGTM! 👍

**Review by heapwolf — CHANGES_REQUESTED** [review]
As @sallar pointed out, your editor has made a lot of whitespace changes. Other than that, the actual fix looks good.

**Comment by sunilhari** [comment]
will do it right away.

**Comment by sunilhari** [comment]
Have changed the indentation.Actually have kept indentation as it as and added the fix like @sallar  suggested.

But not sure how it got merge conflict.When i have submitted PR it had no conflicts.

**Comment by sunilhari** [comment]
Will raise a new pull request incorporating changes.Some of the changes made in recent commits to the base code is bringing null value in the`result` variable which is throwing errors.

**Pull request closed without merging**